### PR TITLE
Gameboard

### PR DESCRIPTION
**Pull request by SiimonStark**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

`Created Board Component w/ mockData, started IntelInput`

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`

## Has This Been Tested?

- [X] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`
- [ ] Have any prior tests been changed?
If so, please explain:
`Your message here`  

# Additional notes:
`Your message here`

# Checklist:

- [ ] My code has no unused/commented out code
- [ ] My code has no console logs
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

Co-Authored-By: Justin Pyktel <justinpyktel@gmail.com>
Co-Authored-By: Lynne Rang <lynne.rang@gmail.com>

### CODE REVIEW

**Comment by lynnerang** [comment]
Looks awesome, great and fast work on the board!  I love that you got intelinput started already and have the words in there starting to look real :).  Also looks like good dimensions for the board, I might slim down the scores lower than the figma dimensions so they're optimized for our macbook aspect ratio.

I have a few thoughts for further tweaks (I'd be happy to incorporate if peeps agree):

1. Can we put the agent status line within the grid cell that holds the board so it is centered with the board/page?  Would also have the benefit of looking like a part of the gameplay as opposed to the header.

2. Can we add an equal width grid gap around the outside of the cards to match the space between the cards?  

Thanks so much for your hard work!

File: src/components/AgentInput/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+
+class AgentInput extends Component {
```
Comment (rdren0) [inline src/components/AgentInput/index.js:3]: One of the components I created would be for this. So we’ll just need to move some of that code into this file

File: src/components/Main/index.js [hunk 2]
```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import Score from '../Score';
+import Board from '../Board';
+import AgentInput from '../AgentInput';
 
 const Main = () => {
```
Comment (rdren0) [inline src/components/Main/index.js:6]: Is Main the container essentially ?

File: src/scss/index.scss [hunk 3]
```diff
@@ -1,18 +1,21 @@
 @import './normalize.scss';
-@import './components/App/App';
-@import './components/Header/Header';
-@import './components/Main/Main';
-@import './components/Score/Score';
+@import './variables.scss';
```
Comment (rdren0) [inline src/scss/index.scss:2]: I also made a variables file, but its in the folder variables and named index.js its NOT for CSS variables, so you dont have to worry about merging them with this doc

**Review by rdren0 — APPROVED** [review]

**Pull request merged**